src/lib/trust_system/peer_status.ml

```diff
@@ -1,17 +1,14 @@
 open Core_kernel
 
+[%%versioned
 module Stable = struct
   module V1 = struct
-    module T = struct
-      type t = {trust: float; banned: Banned_status.Stable.V1.t}
-      [@@deriving bin_io, yojson, version]
-    end
+    type t = {trust: float; banned: Banned_status.Stable.V1.t}
```

Comment:
Use the ppx for `Banned_status` as well?

Reply:
@enolan Can't do that yet because of the `version {asserted}`. There will a similar ppx to handle those cases, coming soon.

Reply:
OK sounds good